### ISSUE TITLE
Mapper Unknown lvalue '--blacklists' in section 'Service' [Witherspoon]

### ISSUE THREAD

**Issue by gkeishin** [openbmc/openbmc#1997]
```
Jul 19 13:04:43 witherspoon systemd[1]: [/lib/systemd/system/xyz.openbmc_project.ObjectMapper.service:9] Unknown lvalue '--blacklists' in section 'Service'
```
```
root@witherspoon:~# cat /lib/systemd/system/xyz.openbmc_project.ObjectMapper.service
[Unit]
Description=Phosphor DBus Service Discovery Manager
Before=obmc-mapper.target

[Service]
Restart=always
Type=dbus
ExecStart=/usr/sbin/phosphor-mapper \
        --path_namespaces="${MAPPER_NAMESPACES}" \
        --interface_namespaces="${MAPPER_INTERFACES}"
        --blacklists="${MAPPER_BLACKLISTS}" \
        --interface_blacklists="${MAPPER_INTERFACEBLACKLISTS}"
BusName=xyz.openbmc_project.ObjectMapper
TimeoutStartSec=300
EnvironmentFile=/etc/default/obmc/mapper
Environment="PYTHONUNBUFFERED=1"

[Install]
WantedBy=obmc-standby.target
root@witherspoon:~#

```

**Comment by gkeishin** [comment]
https://gerrit.openbmc-project.xyz/#/c/5407/

**Comment by sivassrr** [comment]
@gkeishin Are we seeing it in the latest master "Build #899-4403eb0a631434 (Jul 20, 2017 4:14:14 PM) "

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/5407> Fix mapper args syntax error
Resolves  #1997 Mapper Unknown lvalue '--blacklists' in section 'Service' [Witherspoon]
